flatland/envs/rail_env.py

```diff
@@ -192,8 +192,6 @@ def __init__(self,
         self.num_resets = 0
         self.distance_map = DistanceMap(self.agents, self.height, self.width)
 
-        self.action_space = [5]
```

Comment:
Is this removal safe? Remove from `flatland.core.Environment.env` as well or redefine there?